### ISSUE TITLE
BSOD BAD_POOL_CALLER on Windows 10 x64 Pro

### ISSUE THREAD

**Issue by poddex** [ValdikSS/GoodbyeDPI#74]
Hello. Have a trouble with GoodBye DPI 0.1.5rc1 (WinDivert64.sys  ver. 1.3) on Windows 10 x64 Pro. Immediately after start with "-4" or after some time (10-20 min) become a BSOD with BAD_POOL_CALLER.
Using: Firefox browser, Google Chrome. Firewall - ESET Internet Security 11.
In BlueScreenView dump I see errors with NETIO.sys, fwckplnt.sys, Ndu.sys and ntoskrnl.sys. The problem drivers are Ndu.sys and NETIO.sys. How can I solve it?
Already have a 3 BSODs in 2 days.
![bsod](https://user-images.githubusercontent.com/35988134/39087428-404e52f8-45ba-11e8-8920-fd8e568e9d25.jpg)

**Comment by ValdikSS** [comment]
That's probably a bug in your antivirus software. Please report it to the developer, tell them that you're using Windows Filtering Platform via WinDivert driver.

**Comment by catlion** [comment]
Aren't you on Insiders channel by any chance? There is a bug fixed in latest update https://www.computerworld.com/article/3268913/microsoft-windows/microsoft-holds-up-windows-10-feature-upgrade-after-finding-bsod-bug.html

**Comment by poddex** [comment]
GDPI works perfectly for 3-4 days. BSODs begins after 3-4 days of GDPI installation.
And I haven't these MS updates, so this is not an MS-related problem.

**Comment by ValdikSS** [comment]
@poddex did you report the issue to ESET?

**Comment by poddex** [comment]
Reported today. Waiting for a response.

**Comment by poddex** [comment]
@ValdikSS , I got response from ESET. They tell me to 1) uninstall firewall with their brend uninstall utility 2) repair Windows file with WinRepair and 3) Reinstall EIS. I will try these steps in 1-2 days.

**Comment by ValdikSS** [comment]
@poddex, any updates?

**Comment by poddex** [comment]
Sorry, I'm in holiday now.
So, I tested new EIS installation (does it by ESET instruction) for only 2
days before I go to holidays. All works fine (winrepair didn't find any
errors in Windows), GDPI works great. I wonder why it suddenly starts to
bring me a BSODs. Maybe I shutted down my computer when it was in use? Now
I strictly watched it and exited out GDPI before shut down - all work fine
for 5, 10 min, 1 hour, 2 hours and for all browsers.
Thanks.

вс, 6 Май 2018 г., 23:26 ValdikSS <notifications@github.com>:
